source/modules/MEI.cmn.xml

```diff
@@ -797,6 +797,9 @@
             <desc xml:lang="en">Meter signatures with different unit values are used to express a complex metrical
               pattern that is not expressible using traditional means, such as 2/4+1/8.</desc>
           </valItem>
+          <valItem ident="other">
+            <desc xml:lang="en">Meter signatures with different unit values in an unspecified relationship.</desc>
```

Comment:
I am having issues to understand what that means without a look into the corresp. issue. Do you want the relationship explicitly to be "unspecified", or is it just a fallback for all cases not specified above?

Reply:
It is a fallback. Could you propose a better description

Reply:
Sorry @musicEnfanthen, my comment was meant to be a question. Having looked at the issue, could you imagine a better description?

Reply:
No problem @lpugin. I see that it is quite tricky to express without using something like "above", etc. Maybe:

`Meter signatures with different unit values in a relationship not covered by alternating, interchanging or mixed values.` 

(But maybe @ahankinson or @pe-ro can help with a better wording?)

Reply:
How about `Meter signatures in a relationship not covered by the values alternating, interchanging or mixed."?